src/index.ts

```diff
@@ -495,7 +495,7 @@ async function createMainWindow() {
     }
   });
 
-  win.webContents.loadURL(urlToLoad);
```

Comment:
This is intentional behavior: using the `await` keyword prevents the `did-load-failed` event from being called as expected.